src/overlays/actors/ovl_En_Hy/z_en_hy.c

```diff
@@ -9,15 +15,24 @@ void EnHy_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnHy_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnHy_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-extern UNK_TYPE D_06000BFC;
-extern UNK_TYPE D_06000FE4;
-extern UNK_TYPE D_060012E8;
-extern UNK_TYPE D_06002160;
-extern UNK_TYPE D_0600265C;
-extern UNK_TYPE D_060033B4;
+void func_80A70E34(EnHy* this, GlobalContext* globalCtx);
+void func_80A7134C(EnHy* this, GlobalContext* globalCtx);
+void func_80A71530(EnHy* this, GlobalContext* globalCtx);
+void func_80A711B4(EnHy* this, GlobalContext* globalCtx);
+void func_80A712C0(EnHy* this, GlobalContext* globalCtx);
+void func_80A710F8(EnHy* this, GlobalContext* globalCtx);
+void func_80A7127C(EnHy* this, GlobalContext* globalCtx);
+void func_80A712B4(EnHy* this, GlobalContext* globalCtx);
+void func_80A714C4(EnHy* this, GlobalContext* globalCtx);
+
+extern AnimationHeader D_06000BFC;
+extern AnimationHeader D_06000FE4;
+extern AnimationHeader D_060012E8;
+extern AnimationHeader D_06002160;
+extern AnimationHeader D_0600265C;
+extern AnimationHeader D_060033B4;
 extern UNK_TYPE D_06005BC8;
```

Comment:
This symbol can be typed as a display list based on its usage, and you can also omit the `&` where it's used, since it's an array type.
```suggestion
extern Gfx D_06005BC8[];
```